icons/trustwallet.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Trust Wallet</title><path d="M.8175 4.095C0 5.7 0 7.8 0 12s0 6.3.8175 7.905a7.5217 7.5217 0 003.2775 3.2775C5.7 24 7.8 24 12 24s6.3 0 7.905-.8175a7.5216 7.5216 0 003.2775-3.2775C24 18.3 24 16.2 24 12s0-6.3-.8175-7.905A7.5217 7.5217 0 0019.905.8175C18.3 0 16.2 0 12 0S5.7 0 4.095.8175A7.5218 7.5218 0 00.8175 4.095zm17.1713 2.3775c.21 0 .4087.0863.555.2325.1462.15.2287.3525.225.5587-.0375 2.235-.1238 3.945-.285 5.2988-.1575 1.3538-.3938 2.3587-.75 3.1725-.24.5438-.5363.9938-.885 1.3762-.4688.5063-1.005.8738-1.59 1.2226-.2501.1495-.5102.2972-.783.4521-.582.3305-1.2219.694-1.947 1.1791a.7758.7758 0 01-.8663 0c-.7362-.4908-1.3846-.8587-1.972-1.1919a92.1127 92.1127 0 01-.383-.218c-.6862-.3976-1.305-.7763-1.845-1.3239-.36-.36-.6712-.7987-.9187-1.32-.3375-.6975-.5663-1.5412-.735-2.6324-.225-1.4588-.3375-3.3676-.3789-6.015a.7822.7822 0 01.2214-.5588.7952.7952 0 01.5587-.2325h.3225c.9938.0038 3.1875-.0937 5.085-1.5712a.7823.7823 0 01.9563 0c1.8975 1.4775 4.0912 1.575 5.0887 1.5712h.3263zm-1.7438 8.7638c.2438-.5026.4463-1.1963.6-2.1938.1838-1.1925.2963-2.8125.3488-5.0175-1.17-.0338-3.18-.2587-5.0963-1.5487C10.1813 7.7625 8.1713 7.9875 7.005 8.025c.0413 1.8225.1238 3.24.255 4.3538.15 1.2675.3638 2.1262.63 2.73.1763.4012.3713.69.6038.945.3112.3412.705.6225 1.2412.945.2225.1334.4677.2722.7347.4234.4756.2691 1.0202.5774 1.6278.964.5965-.3803 1.1332-.6854 1.6033-.9526.1417-.0805.2773-.1576.4067-.2324.66-.3787 1.1475-.6937 1.5113-1.0612.2437-.2512.4425-.525.6262-.9037z"/></svg>
```

Comment:
I took the fifth icon in SVG from https://trustwallet.com/press to do my comparison:
![image](https://user-images.githubusercontent.com/12817388/122134660-0c3aaa00-ce0d-11eb-9708-96866abc45bc.png)

It looks really different. Would you mind taking another try for this one?

![image](https://user-images.githubusercontent.com/12817388/122134726-312f1d00-ce0d-11eb-8266-acaab4c71034.png)

Reply:
Yes. I think fifth icon is the best for us. I have updated it and compare to the source, it fit perfectly. But notice, the fifth icon we download from the press page is not optimized (the line of shield is still a line format instead of path format, so if we scale it before converting, it will be scale the line width as well.

Reply:
Thank you for your work! This is much better, but I'm still seeing some differences in the inner shield. Ex. 
![image](https://user-images.githubusercontent.com/12817388/122404953-a43fad00-cf4d-11eb-9960-92b46256d24f.png)

Reply:
That could be a result of Inkscape butchering the conversion of the stroke to a path, same thing happened when I tried it. Assuming @huuphongnguyen is using Inkscape, that is.

Reply:
Yes. I'm using Sketch in main and Inkscape to compare. Sometime, Sketch is not good at reading the original file (which was optimized) so I use Inkscape first, after that export it and import to Sketch to center, afjust size, etc. I'm using Sketchapp in main way. 
Export in Sketch and optimize in SVGOMG with Percision=4 is perfectly.
I will check it again.